alvenir/guidance/v1/guidance.proto

```diff
@@ -1,67 +1,73 @@
 syntax = "proto3";
+
 package alvenir.guidance.v1;
 
 
 message LlmPromptAnswer {
-  string prompt = 1;
-  string context = 2;
-  string answer = 3;
+    string prompt = 1;
```

Comment:
Something with indentation where this repo is inconsistent. It seems like someone suggests 2 space indent [1] but I also see in many of the other files that it is mixed 

[1] https://protobuf.dev/programming-guides/style/#standard-file-formatting

Reply:
fixed